instrumentation/opentelemetry-instrumentation-sqlalchemy/tests/test_sqlalchemy.py

```diff
@@ -182,6 +182,17 @@ def test_create_engine_wrapper(self):
             "opentelemetry.instrumentation.sqlalchemy",
         )
 
+    def test_instrument_engine_from_config(self):
+        SQLAlchemyInstrumentor().instrument()
+        from sqlalchemy import engine_from_config  # pylint: disable-all
+
+        engine = engine_from_config({"sqlalchemy.url": "sqlite:///:memory:"})
+        cnx = engine.connect()
+        cnx.execute("SELECT	1 + 1;").fetchall()
```

Comment:
This should fix some current `sqlalchemy.exc.ObjectNotExecutableError` in the tests:

```suggestion
        cnx.execute(text("SELECT	1 + 1;")).fetchall()
```